satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -213,7 +213,7 @@ model user (
     field password_hash    blob      ( updatable )
 
     field status           int       ( updatable, autoinsert )
-
+    field partner_id       blob      ( nullable )
```

Comment:
I told you incorrectly, we don't need the partner_id on the bucket itself since we will be at the point of the value attribution logic from the first part you did.